packages/editor-ui/src/composables/useDebugInfo.ts

```diff
@@ -96,7 +96,7 @@ export function useDebugInfo() {
 	};
 
 	const pruningInfo = () => {
-		if (!settingsStore.pruning.isEnabled) return { enabled: false } as const;
+		if (!settingsStore.pruning?.isEnabled) return { enabled: false } as const;
```

Comment:
Is the TS type correct for `settingsStore.pruning`? Could we have prevented this with better typing?

Reply:
Types are correct, but after pinia unrefs everything, `useSettingsStore` seems to have incorrect types on some occasions.

Reply:
Is there anything we can do about that?

Reply:
Found the culprit. Fixed the types and added `@n8n/api-types` to `tsconfig.json` to pick up changes without building.